lighthouse-core/report/v2/report-template.html

```diff
@@ -30,8 +30,11 @@
   <script>%%LIGHTHOUSE_JAVASCRIPT%%</script>
   <script>window.__LIGHTHOUSE_JSON__ = %%LIGHTHOUSE_JSON%%;</script>
   <script>
-    const renderer = new ReportRenderer(document);
-    document.body.appendChild(renderer.renderReport(window.__LIGHTHOUSE_JSON__));
+    const dom = new DOM(document);
```

Comment:
Ha, DI! I feel like we'll have to move more implementation out as time goes on. I guess it's fine for now.